### ISSUE TITLE
Domain of x

### ISSUE THREAD

**Issue by gmikel** [uwhpsc-2016/homework3#4]
I noticed that the implementation of the test specifies a linear space while the definition given for both the Trapezoidal and Simpson's Rules are not necessarily defined relative to a linear space (i.e. the step size isn't explicitly defined). 

Can we assume that only linear spaces are going to be passed to our functions or should we implement a more "general" case whereby a step size is calculated for each iteration?

**Comment by gmikel** [comment]
Nevermind, I missed the sentence at the top of the "Tests" definition -- "Your implementations will be run through the following test suite. Tests will use an x where distance between subsequent points, x[i+1]-x[i], is not necessarily uniform."

**Comment by cswiercz** [comment]
> Can we assume that only linear spaces are going to be passed to our functions or should we implement a more "general" case whereby a step size is calculated for each iteration?

No. You are correct in identifying that some of the example code in the `README` use `numpy.linspace` which produces an array with a uniform `dx`. However, we will be testing your code on `x`-data that does not necessarily satisfy this property.
